lib/repl.js

```diff
@@ -698,6 +699,10 @@ function filteredOwnPropertyNames(obj) {
   return Object.getOwnPropertyNames(obj).filter(intFilter);
 }
 
+REPLServer.prototype.complete = function() {
```

Comment:
@lance I'm not sure about its origins or all its purposes (besides the one related to testing), since it was already there when I added the feature, I just updated it so nothing is broken.